src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
+
+  let sum = 0;
+
+  men.forEach((m) => sum += m.died - m.born);
+
+  return sum/men.length;
```

Comment:
1. I recommend creating additional variables for intermediate results and choosing the right naming for them.
2. Assignment comment says to practice with methods like .filter .map .some .every .find .reduce
To calculate the sum of the total age of men, the .reduce method is well suited. [About .reduce method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)
3. Use spaces below '/' . This will be a good practice, I recommend checking out [Airbnb JavaScript Style Guide](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)

I propose to slightly correct your code following my points:
```suggestion
   const sumAge = men.reduce((accumulator, currentMen) => accumulator + currentMen.died - currentMen.born, 0);

  const menCount = men.length
  const menAverageAge = sumAge / menCount 

  return menAverageAge;
```